packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -70,6 +75,21 @@ export class TaskItem extends Component {
   }
 }
 
+const taskObjectsRenderer = ({ objects }) =>
+  map(objects, obj => <p>{renderXoItem(obj, { link: true })}</p>)
```

Comment:
`renderXoItem` doesn't return a paragraph, IMHO, the usage of the tag `p` isn't appropriate.  You can use a list instead.